### PR TITLE
Require ext

### PR DESCRIPTION
**Pull request by nickpell**
@dannyroberts

### CODE REVIEW

File: soil/tasks.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def prepare_download(download_id, payload_func, content_disposition,
         payload = payload_func(process=prepare_download)
     except TypeError:
         payload = payload_func()
-    expose_cached_download(payload, expiry, mimetype=content_type,
+    expose_cached_download(payload, expiry, None, mimetype=content_type,
                            content_disposition=content_disposition,
                            download_id=download_id)
```
Comment (dannyroberts) [inline soil/tasks.py]: Does this end up being the same as `file_extension='.xls'`? If so, I think I'd prefer making that explicit here and removing the default.

Comment (dannyroberts) [inline soil/tasks.py]: (And asserting that file_extension is not None and either failing hard or soft asserting.)

Comment (nickpell) [inline soil/tasks.py]: :+1:

Comment (nickpell) [inline soil/tasks.py]: Offlined; not appending a default file extension, but making it an explicit argument in functions where it should be passed.

**Pull request merged**